rocAL/rocAL_pybind/amd/rocal/fn.py

```diff
@@ -277,15 +276,15 @@ def resize(*inputs, bytes_per_sample_hint=0, image_type=0, interp_type=1, mag_fi
     """
 
     # pybind call arguments
-    kwargs_pybind = {"input_image0": inputs[0], "dest_width": resize_x, "dest_height": resize_y,
-                     "is_output": False, "scaling_mode": scaling_mode, "max_size": max_size, "resize_shorter": resize_shorter, 
-                     "resize_longer": resize_longer, "interpolation_type": interpolation_type }
+    kwargs_pybind = {"input_image0": inputs[0], "dest_width": resize_x , "dest_height": resize_y, "is_output": False, "scaling_mode": scaling_mode, "max_size": max_size, "resize_shorter": resize_shorter, 
+                     "resize_longer": resize_longer, "interpolation_type": interpolation_type, "output_layout" : output_layout, "output_dtype" : output_dtype}
     resized_image = b.Resize(Pipeline._current_pipeline._handle ,*(kwargs_pybind.values()))
     return (resized_image)
 
-def resize_mirror_normalize(*inputs, bytes_per_sample_hint=0, resize_min=0, resize_max=0, 
-                            image_type=0, mean=[0.0], mirror=1, output_dtype=types.FLOAT, output_layout=types.NCHW, pad_output=False,
-                            preserve=False, seed=1, std=[1.0], device=None):
+def resize_mirror_normalize(*inputs, bytes_per_sample_hint=0, interp_type=1, mag_filter= 1, max_size = [], min_filter = 1, minibatch_size=32,
```

Comment:
Remove extra space after mag filter argument and max size. Either add space between = or not consistently for all params.

Reply:
Done